analytics-datastore-elastic-search/swarm.sh

```diff
@@ -1,172 +1,217 @@
 #!/bin/bash
 
-ACTION=$1
-MODE=$2
+declare ACTION=""
+declare MODE=""
+declare COMPOSE_FILE_PATH=""
+declare ROOT_PATH=""
+declare container_status=""
+declare service_names=()
+
+function init_vars() {
+  ACTION=$1
+  MODE=$2
+
+  COMPOSE_FILE_PATH=$(
+    cd "$(dirname "${BASH_SOURCE[0]}")" || exit
+    pwd -P
+  )
+
+  ROOT_PATH="${COMPOSE_FILE_PATH}/.."
+
+  if [[ "${NODE_MODE}" == "cluster" ]]; then
+    container_status="Running"
+    service_names=(
+      "analytics-datastore-elastic-search-01"
+      "analytics-datastore-elastic-search-02"
+      "analytics-datastore-elastic-search-03"
+    )
```

Comment:
I think to declare and do the iteration will be more code and more comlexity than just declaring them? it will look like this: 
![image](https://user-images.githubusercontent.com/48720958/207876196-1160e02f-a7e1-4ea1-b1da-e7a721b0d880.png)

If you still think that iteration is better I can update it

Reply:
```suggestion
    #!/bin/bash
for i in {1..3}
do
   # ...
done
```

Reply:
How about this ?